pkgs/applications/office/timekeeper/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, fetchFromGitHub
+, poco
+, pkgconfig
+, gnome2
+, lib
+}:
+
+stdenv.mkDerivation rec {
+  pname = "timekeeper";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "bburdette";
+    repo = "TimeKeeper";
+    rev = "363a3f0033fcca2d45b0d1f0fc74d6470c9db10d";
+    fetchSubmodules = true;
+    sha256 = "03alrfy7axq64k9lrs14afc0m6zn5gr5glhgx9i2r1yam7wdmz5c";
+  };
+
+  buildInputs = [
+    poco
+    pkgconfig
+  ] ++ (with gnome2; [
+    gtkmm
```

Comment:
Also is it not possible to build the app against `gtkmm3`? GTK 2 and libglade has been deprecated for ages.

Reply:
I having worked on this code in about 10 years.  I wanted to just compile it and make it available, not rewrite it.  I'll see if it will work without having to work on the code.  If I have to do a significant rewrite I'll probably bail on publishing.

Reply:
Ok I fixed all the things but not this.  If its really a deal breaker I'll see what I can do.